drivers/crypto/Kconfig

```diff
@@ -128,4 +122,43 @@ endchoice
 
 endif # CRYPTO_ATAES132A
 
+if CRYPTO_SHIM
+
+config CRYPTO_ENABLE_AES
+	bool "Enable AES operations"
+	default y
+
+if CRYPTO_ENABLE_AES
+config CRYPTO_ENABLE_AES_CBC
+	bool "Enable CBC (block) mode for AES operations"
+	default n
```

Comment:
I know Zephyr has this all over, but just a small note: `bool` symbols implicitly default to `n`. `default n` is almost always redundant.